### ISSUE TITLE
aria-label matching doesn't appear to handle some characters

### ISSUE THREAD

**Issue by willhall88** [ThePacielloGroup/ARC-Toolkit#69]
**Describe the bug**
ARC Toolkit doesn't appear to handle some characters, causing false positives. 
I've reproduced with `'` and `,`, but haven't tested beyond that.

**To Reproduce**

1. Create an anchor tag with text that includes special characters in the `text` and `aria-label`  e.g. `<a href="#" aria-label="O'Malley">O'Malley</a>`
2. Use the Arc Toolkit devTools to run tests
3. Observe that an Alert appears: 
![image](https://github.com/user-attachments/assets/938701c9-67f5-4c4a-9d84-586e4e9f146f)
4. Remove the special character from the tag and observe that the Alert disappears.

-----
Test Case (CodePen/JSBin/etc.): https://codepen.io/willhall88/pen/xbKZJzW?editors=1000

**Expected behavior**
An `a` tag that contains the visible text in the aria-label should not produce any accessibility alerts

**Version information**

- Browser and version: Chrome Version 131.0.6778.86
- ARC Toolkit version: 5.7.6

**Comment by ghost** [comment]
Yes, I was able to reproduce this with the `/` character, and it throws a false positive error.

**Comment by ferllings** [comment]
Thanks @vickydass a fix will be released soon.

**Comment by poorgeek** [comment]
Fixed in ARC Toolkit 5.7.9

**Comment by ghost** [comment]
Confirmed. Thank you @poorgeek & @ferllings
